### PR TITLE
weshoulddeletethe login page

### PR DESCRIPTION
**Pull request by zzcipod**
debug

### CODE REVIEW

**Comment by zzcipod** [comment]
?

**Review by zzcipod — COMMENTED** [review]
debug

**Review by zzcipod — COMMENTED** [review]
000

**Comment by zzcipod** [comment]
111

File: client/src/pages/Login.js [hunk 1]
```diff
@@ -22,14 +22,16 @@ const Login = (props) => {
   };
 
   // submit form
+  
   const handleFormSubmit = async (event) => {
+  debugger
```
Comment (prsoulman) [inline client/src/pages/Login.js]: please take out debugger

File: client/src/utils/auth.js [hunk 2]
```diff
@@ -29,7 +29,7 @@ class AuthService {
 
   login(idToken) {
     localStorage.setItem('id_token', idToken);
-    window.location.assign('/');
+    window.location.assign('/selectMovies');
```
Comment (prsoulman) [inline client/src/utils/auth.js:32]: This is approved

**Review by prsoulman — CHANGES_REQUESTED** [review]

**Pull request closed without merging**